phys2bids/interfaces/mat.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""phys2bids interface for MATLAB files."""
+
+import logging
+
+import numpy as np
+from scipy.io import loadmat
+
+from phys2bids.physio_obj import BlueprintInput
+
+LGR = logging.getLogger(__name__)
+
+
+def populate_phys_input(filename, chtrig):
+    """
+    Populate object phys_input from MATLAB files.
+
+    Parameters
+    ----------
+    filename: str
+        path to the txt labchart file
+    chtrig : int
+        index of trigger channel.
+        !!! ATTENTION: IT'S MEANT TO REPRESENT AN INDEX STARTING FROM 1 !!!
+
+    Returns
+    -------
+    BlueprintInput
+
+    Note
+    ----
+    chtrig is not a 0-based Python index - instead, it's human readable (i.e., 1-based).
+    This is handy because, when initialising the class, a new channel corresponding
+    to time is added at the beginning - that is already taken into account!
+
+    See Also
+    --------
+    physio_obj.BlueprintInput
+    """
+    # Load MATLAB file into dictionary.
+    mat_dict = loadmat(filename)
+
+    # Convert data into 1d numpy array for easier indexing.
+    data = np.squeeze(np.asarray(mat_dict['data']))
+
+    # Extract number of channels and tick rate.
+    n_channels = len(mat_dict['titles'])
+    t_freq = mat_dict['tickrate'][0][0]
+
```

Comment:
Here you are assuming t_freq is always in Hz. Are you sure this is the case always?

Reply:
Well, I didn't have any other info in the MATLAB files that we have in OSF. So I had to assume the frequencies are given in Hz (they definitely look like they're in Hz anyway).

Is this always the case @RayStick @smoia @tsalo ?

Reply:
I don't know if this is always the case (have not used this .mat formats before, generated from a labchart export) but I would assume it be Hz considering that is the units LabChart uses. I guess a .mat file that is not generated from LabChart could be different

Reply:
Well I suppose that's something we have to specify then

Reply:
We should see if for example ``acq`` files can be in matlab format and add it to this function

Reply:
![imagen](https://user-images.githubusercontent.com/38909338/91276964-7f8e0500-e782-11ea-9f15-75f6d1d8b609.png)
Seems like acq does have matlab format. Therefore we can either leave it for another PR or add it in this one. In any of those case I would put a condition in the function to check that the ``.mat`` file is only in labchart format

Reply:
What do you mean by "disregarding any other structure"? Do you only want labchart MATLAB files to be supported?

Reply:
So if in this PR you only support the labchart structure, yes i want an if condition that checks something that you think is specific of labchart matlab files. In case you are gonna implement also de acq MATLAB files then accept only those two. Basically i want you to check that the ``.mat`` file people load is not some random MATLAB file

Reply:
Mainly because this way we can raise a error